api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistMediaRepository.kt

```diff
@@ -42,7 +40,6 @@ class AnilistMediaRepository @Inject constructor(
                     seasonYear = Optional.presentIfNotNull(seasonYear)
                 )
             )
-            .fetchPolicy(FetchPolicy.CacheAndNetwork)
             .toFlow()
```

Comment:
Using `watch()` fixes observability. For refreshing the cache, I think our best bet for now is to make a separate query that runs `NetworkOnly`, which should update the cache with fresh data. We don't have to use the result, but it might be nice to know if it succeeded or failed

Reply:
I guess this isn't happening now